romancal/regtest/test_dark_current.py

```diff
@@ -0,0 +1,82 @@
+"""Regression test for the Dark current subtraction step."""
+import pytest
+
+from romancal.stpipe import RomanStep
+from .regtestdata import compare_asdf
+
+
+@pytest.mark.bigdata
+def test_dark_current_subtraction_step(rtdata, ignore_asdf_paths):
+    """ Function to run and compare Dark Current subtraction files. Note: This
+        should include tests for overrides etc. """
+    rtdata.get_data(
+        "WFI/image/r0019101001023012008_01101_0173_WFI01_jumpstep.asdf")
+    rtdata.input = "r0019101001023012008_01101_0173_WFI01_jumpstep.asdf"
+
+    args = ["romancal.step.DarkCurrentStep", rtdata.input]
+    RomanStep.from_cmdline(args)
+    output =\
+        "r0019101001023012008_01101_0173_WFI01_jumpstep_darkcurrentstep.asdf"
+    rtdata.output = output
+    rtdata.get_truth(f"truth/WFI/image/{output}")
+    assert (compare_asdf(rtdata.output, rtdata.truth,
+            **ignore_asdf_paths) is None)
+
+
+@pytest.mark.bigdata
+def test_dark_current_outfile_step(rtdata, ignore_asdf_paths):
+    """ Function to run and compare Dark Current subtraction files. Here the
+        test if for renaming the output file. """
+    rtdata.get_data(
+        "WFI/image/r0019101001023012008_01101_0173_WFI01_jumpstep.asdf")
+    rtdata.input = "r0019101001023012008_01101_0173_WFI01_jumpstep.asdf"
+
+    args = ["romancal.step.DarkCurrentStep", rtdata.input,
+            '--output_file=Test_dark']
+    RomanStep.from_cmdline(args)
+    output = "Test_dark_darkcurrentstep.asdf"
+    rtdata.output = output
+    rtdata.get_truth(f"truth/WFI/image/{output}")
+    assert (compare_asdf(rtdata.output, rtdata.truth,
+            **ignore_asdf_paths) is None)
+
+
+@pytest.mark.bigdata
+def test_dark_current_outfile_suffix(rtdata, ignore_asdf_paths):
+    """ Function to run and compare Dark Current subtraction files. Here the
+        test if for renaming the output file. """
```

Comment:
if -> is, though I think this comment should be change to indicate that it is the suffix renaming that is being tested.